src/Confluent.Kafka/Consumer.cs

```diff
@@ -620,6 +620,21 @@ public Metadata GetMetadata(bool allTopics, TimeSpan timeout)
         /// </summary>
         public Metadata GetMetadata(bool allTopics)
             => consumer.GetMetadata(allTopics);
+
+        /// <summary>
+        ///     Adds one or more brokers to the kafka bootstrap brokers.
```

Comment:
More verbose summaries is what's blocking me LGTMing this PR - feel free to have another go at this, or I'll get to it myself later.